content/en/docs/ops/configuration/mesh/config-resource-ready/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Wait for Resource Status to Apply Configuration
+description: Describes how to wait to apply mesh configuration until a resource reaches a given status or readiness.
+weight: 15
+---
+
+{{< warning >}}
+This feature is in the Alpha stage, see
+[Istio Feature Status](/about/feature-stages/). Your feedback is welcome in the
+[Istio User Experience discussion](https://discuss.istio.io/c/UX/23). Currently,
+this feature is tested only for single, low volume clusters with a single
+control plane revision.
+{{< /warning >}}
+
+Istio's mesh configuration is declarative, which means that you define a
+configuration and Istio propagates the changes through the mesh over time. As a
+result, Istio might attempt to apply consecutive configuration changes before
+the target resources are ready.
+
+In Istio 1.6 and later, you can use the `kubectl wait` command to have more
+control over the way that Istio applies configuration changes to the mesh. To
+make this possible, the `kubectl wait` command monitors the
+[`status` field](/docs/reference/config/config-status/) of the resource's
+[`CustomResourceDefinition`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+(CRD), which Istio updates as it propagates configuration changes.
+
+## Before you begin
+
+This feature is off by default. Enable the `status` field as part
+of Istio installation. If you enable it after installation, you must re-deploy
+the control plane.
+
+## Wait for resource readiness
+
+You can apply a change to a virtual service and then wait for completion, using
+the following commands:
+
+{{< text bash >}}
+$ kubectl apply -f virtual_service_name.yaml
+$ kubectl wait --for=condition=Reconciled virtual_service/name
+{{< /text >}}
+
+This blocking command does not release until the previous command
+completes or reaches a time out.
```

Comment:
"until the virtual service has been distributed to all proxies in the mesh, or until the command times out."

Reply:
fixed